README.md

```diff
@@ -6,10 +6,12 @@ and to generate graphs in a running instance.
 
 # TL;DR:
 
-- importing a Graphalytics graph:
+- importing a [Graphalytics](https://graphalytics.org/) graph:
 
 ```commandline
-python importer.py http://localhost:8529/_db/_system graphalytics --dir_graphalytics /PATH/GRAPH_DIRECTORY 
+wget https://atlarge.ewi.tudelft.nl/graphalytics/zip/wiki-Talk.zip
+unzip wiki-Talk.zip
+python importer.py http://localhost:8529/_db/_system graphalytics --dir_graphalytics wiki-Talk
```

Comment:
TODO: test this even works (at the time of writing, the graphalytics data server is not reachable.)

Reply:
Instead of this hack, just give a call to `start_Pregel.py`